### PR TITLE
Add option `g:manpager_default_keymaps`

### PR DESCRIPTION
**Pull request by sainnhe**
This configuration option allows users to disable default key mappings and define their own.

Related issues: #18

### CODE REVIEW

File: doc/vim-manpager.txt [hunk 1]
```diff
@@ -102,6 +103,31 @@ COMMAND					*vim-manpager-command*
 	under the cursor will be used.
 	It overwrite the default |:Man| command to improve the behavior.
 
+==============================================================================
+CONFIGURATION				*vim-manpager-configuration*
+
+
+					*g:manpager_default_keymaps*
```
Comment (lambdalisue) [inline doc/vim-manpager.txt]: Please make it like `g:manpager_disable_default_mappings` and make it default `0`

**Review by lambdalisue — CHANGES_REQUESTED** [review]
Thanks for PR. Please follow my comment.

File: doc/vim-manpager.txt (hunk 1 above)
Comment (sainnhe) [reply doc/vim-manpager.txt]: Thanks! I've forced push a commit to rename this option.

**Review by lambdalisue — APPROVED** [review]
LGTM

**Pull request merged**